### DISCUSSION TITLE
Using docker-slim for third-party image like airbyte, airflow

### DISCUSSION THREAD

**Discussion by rajeshjatavkonvergeai** [slimtoolkit/slim#259] [Q&A]
Would docker-slim also work for third-party images like airbyte? If yes, then how to slim it. Actually, I tried to slim it by docker-slim. I got the following error
![docker-slim_error2](https://user-images.githubusercontent.com/82082215/147067209-1a4ea4d0-89cd-4e18-a376-03ba21c93711.jpg)

So please suggest how to slim third-party images?

**Comment by kcq** [comment]
@rajeshjatavkonvergeai the airbyte/worker doesn't have any network ports declared in its Dockerfile, which is often the case with the CLI apps. Makes sense that the airbyte worker is not a server app :) For the CLI apps you need to disable http probing. That's one of the options mentioned in the error message, so you'd need to run DS like this: `docker-slim build --http-probe=false airbyte/worker:YOUR_TAG`
